test/CMakeLists.txt

```diff
@@ -77,9 +77,9 @@ teca_add_test(test_cf_reader_era5
 
 teca_add_test(test_cf_reader_file_time
     COMMAND test_cf_reader
-    -i "${TECA_DATA_ROOT}/ARTMIP_MERRA_2D_20170210_06\.nc"
```

Comment:
We need to keep this test as is because the test tests getting time dimension from the file name. In the new dataset the time dimension is encoded in the file itself rather than the file name.